schemaValidator.ts

```diff
@@ -45,9 +47,17 @@ const latestMetadataSchema = "https://schemas.premid.app/metadata/1.1",
   for (const metaFile of metaFiles) {
     const meta = loadMetadata(metaFile),
       service = meta.service,
-      result = validate(meta, schema);
+      result = validate(meta, schema),
+      validLangs = (await axios.get("https://api.premid.app/v2/langFile/list"))
+        .data,
+      invalidLangs: string[] = [];
+
+    Object.keys(meta.description).forEach((lang) => {
+      const index = validLangs.findIndex((l: string) => l == lang);
+      if (index == -1) invalidLangs.push(lang);
+    });
 
-    if (result.valid) {
+    if (result.valid || !invalidLangs.length) {
```

Comment:
|| should be &&

Gotta love not being able to sleep because I am think back code lmao

Will fix tomorrow or @ririxidev I think you can edit it too